lib/core/src/lib/viewer/components/viewer.component.spec.ts

```diff
@@ -525,6 +525,33 @@ describe('ViewerComponent', () => {
             });
         });
 
+        describe('Close Button', () => {
+
+            it('should show close button on left side when closeButtonPosition is left and allowGoBack is true', () => {
```

Comment:
Could you additionally check if in that case close button is not displayed at right side?